File: server/medicore.layer/MediCore.Service/Model/MedicalStaffImageModel.cs
using dna.core.service.Models;
using dna.core.service.Models.Abstract;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MediCore.Service.Model
{
    public class MedicalStaffImageModel : IModelBase
    {
        public int Id { get; set; }

        [Required]
        public int MedicalStaffId { get; set; }

        [Required]
        public int ImageId { get; set; }

      
        public MedicalStaffModel MedicalStaff { get; set; }

        
        public ImageModel Image { get; set; }
    }
}

File: smartmedika.admin/src/app/model/medicalStaffImageModel.ts
import { IEntityBase } from './IEntityBase';
import { MedicalStaffModel } from './medicalStaffModel';
import { ImageModel } from './imageModel';

export class MedicalStaffImageModel implements IEntityBase{
    id:number
    medicalStaffId:number;
    imageId:number;
    medicalStaff:MedicalStaffModel;
    image:ImageModel;
}